python/dgl/_deprecate/frame.py

```diff
@@ -0,0 +1,1018 @@
+"""Columnar storage for DGLGraph."""
```

Comment:
Is this module still in use?

Reply:
Will delete these `_deprecate` files later.